gluegun.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.15"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_runtime_dependency 'github-markup', "~> 0"
```

Comment:
Shouldn't the dependency be github-markdown instead of github-markup? I remember not being able to solve the problem when I installed this dependency instead of github-markdown